frontend/lib/api/election/ElectionStatusProvider.tsx

```diff
@@ -14,20 +15,20 @@ export interface ElectionStatusProviderProps {
 }
 
 export function ElectionStatusProvider({ children, electionId }: ElectionStatusProviderProps) {
-  const { data, loading, error } = useElectionStatusRequest({
+  const { data, error, refetch } = useElectionStatusRequest({
     election_id: electionId,
   });
 
-  if (loading) {
+  if (data === null) {
     return null;
   }
 
-  if (!data || error) {
-    throw new Error();
+  if (error) {
```

Comment:
shouldn't this be defined above `if (data === null) return null;` ? (just in case)

Reply:
I'll create a PR to fix that!

Reply:
#354